apps/api-gateway/src/core/config/config.service.ts

```
@@ -0,0 +1,12 @@
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+
+@Injectable()
+export class ConfigXService {
+  constructor(private readonly cfg: ConfigService) {}
+
+  get natsUrl(): string { return this.cfg.get<string>('NATS_URL')!; }
```

ESLint

[error] Replace `·return·this.cfg.get<string>('NATS_URL')!;` with `⏎····return·this.cfg.get<string>('NATS_URL')!;⏎·`

(prettier/prettier)